src/zoning-district/zoning-district.repository.schema.ts

```diff
@@ -1,6 +1,21 @@
-import { zoningDistrictEntitySchema } from "src/schema";
+import {
+  zoningDistrictClassEntitySchema,
+  zoningDistrictEntitySchema,
+} from "src/schema";
 import { z } from "zod";
 
+export const checkByIdRepoSchema = zoningDistrictEntitySchema.pick({
+  id: true,
+});
+
+export type CheckByIdRepoSchema = z.infer<typeof checkByIdRepoSchema>;
```

Comment:
The `schema` suffix is used only on zod schemas

```suggestion
export type CheckByIdRepo = z.infer<typeof checkByIdRepoSchema>;
```